controls/def.cf

```diff
@@ -126,6 +126,14 @@ bundle common def
                           not(isvariable("trustkeysfrom"))),
         meta => { "defvar" };
 
+      "cf_serverd_maxconnections"
```

Comment:
The bundle would allow us to specify default values in the MPF that may be more appropriate to general use than the defaults in C.

With the single augments file we currently have only specifying values would be limiting.

That bundle might look something like this:

```
body file control
{
  namespace => "cfengine";
}

bundle common controls
{
  vars:
    "server[maxconnections]" int => ifelse( isvariable("server[maxconnections]"), "$(server[maxconnections])", "200");
}

It's really no different than doing it in def and agreeing on a consistent structure.